packages/mdc-menu/README.md

```diff
@@ -198,9 +198,16 @@ With focus on the menu button:
 
   * <kbd>Enter</kbd>, <kbd>Space</kbd> & <kbd>Down Arrow</kbd> opens the menu and places focus on the first menu item.
   * <kbd>Up Arrow</kbd> opens the menu and moves focus to the last menu item.
-  * The focus is set to menu root element when clicked or touched. Menu handles the keyboard handling once it is opened and focused on root element.
+  * The focus is set to list root element (where `role="menu"` is set) when clicked or touched. MDC List handles the keyboard navigation once it receives the focus.
+
+Use `setDefaultFocusState` method to set default focus state that will be focused every time when menu is opened.
 
-Use `setDefaultFocusItemIndex` method to set the index of the menu item that will be focused every time the menu opens. Set it to `numbers.FOCUS_ROOT_INDEX` to focus on menu root element.
+Focus state | Description
+--- | ---
+`DefaultFocusState.FIRST_ITEM` | Focuses the first menu item. Set this when menu button receives <kbd>Enter</kbd>, <kbd>Space</kbd>, <kbd>Down Arrow</kbd>.
```

Comment:
is kbd the same as the backticks (`)?

Reply:
`kbd` is for keyboard keys, where as backticks is used when referencing code.

![image](https://user-images.githubusercontent.com/47312/56054521-0e80b580-5d25-11e9-9348-63f98f615f2c.png)